.all-contributorsrc

```diff
@@ -1143,7 +1143,15 @@
       "contributions": [
         "doc"
       ]
-    }
+    },
+    {
+      "login": "xloem",
+      "name": "patiently pending world peace",
```

Comment:
... interesting name :-)
I wouldn't mind it if that's what you want

Reply:
My experience is that it is unwise to share our names until things improve, but I don't want to push my views on others.  Happy to change it if you'd like.

Reply:
no, no, all fine - being anonymous is fine, and world peace is a goal I can also identify with.